libbeat/monitoring/logging.go

```diff
@@ -0,0 +1,157 @@
+package monitoring
```

Comment:
I moved this code the logs the metrics out of the logp package. Separation of concerns and all...

I just dropped it in the monitoring package and hacked it to make it work. I was wondering it it would make sense to have a libbeat/monitoring/report/log package that contains this code for reporting the metrics to the log?